docs/contributing/index.md

```diff
@@ -15,37 +15,37 @@ Sumo Logic Documentation is an Open Source project available as a [GitHub reposi
 There are many ways to contribute to Sumo Docs, beyond writing code. Here's how you can get involved.
```

Comment:
You still need to update this article to account for the combined create-edit-doc file.

Reply:
thank you - done!